guides/common/modules/proc_securing-the-dhcp-api.adoc

```diff
@@ -1,12 +1,12 @@
 [id="Securing_the_dhcpd_API_{context}"]
 = Securing the dhcpd API
 
-{SmartProxy} interacts with DHCP daemon using the dhcpd API to manage DHCP.
+{SmartProxyServer} interacts with DHCP daemon using the dhcpd API to manage DHCP.
```

Comment:
This isn't true. It's the Smart Proxy process that communicates with the DHCP daemon. IMHO is SmartProxyServer the whole server.